apps/prairielearn/src/middlewares/AuthzAccessMismatch.tsx

```diff
@@ -158,55 +178,35 @@ export function AuthzAccessMismatch({
   return (
     <main id="content" class="container">
       <div class="card mb-4">
-        <div class="card-header bg-danger text-white">Insufficient access</div>
+        <div class="card-header bg-danger text-white">
+          <h1>Effective user has insufficient access</h1>
+        </div>
         <div class="card-body">
-          <h2 class="mb-3 h4">Effective user has insufficient access</h2>
+          <p>{errorExplanation}</p>
           <p>
-            The
+            The current effective user{' '}
             {authzUser ? (
-              <OverlayTrigger
-                overlay={
-                  <Tooltip>
-                    {authzUser.name} ({authzUser.uid})
-                  </Tooltip>
-                }
-              >
-                <button type="button" class="btn btn-link link-secondary p-0 mx-1 align-baseline">
-                  current effective user
-                </button>
-              </OverlayTrigger>
+              <strong>
+                {authzUser.name} ({authzUser.uid})
+              </strong>
             ) : (
-              ' current user '
-            )}
-            does
-            <OverlayTrigger overlay={<Tooltip>Error message: {errorMessage}</Tooltip>}>
-              <button type="button" class="btn btn-link link-secondary p-0 mx-1 align-baseline">
-                not have access
-              </button>
-            </OverlayTrigger>
-            to this page, but
-            <OverlayTrigger
-              overlay={
-                <Tooltip>
-                  {authnUser.name} ({authnUser.uid})
-                </Tooltip>
-              }
-            >
-              <button type="button" class="btn btn-link link-secondary p-0 mx-1 align-baseline">
-                your account
-              </button>
-            </OverlayTrigger>
+              <strong>your account</strong>
+            )}{' '}
+            does not have access to this page, but your account{' '}
+            <strong>
+              {authnUser.name} ({authnUser.uid})
+            </strong>{' '}
             does.
           </p>
 
           <details class="mb-3">
             <summary>View permission differences</summary>
             <div class="mt-3">
-              <h6>One of these permissions is required</h6>
+              <h3 class="h6">One of these permissions is required</h3>
```

Comment:
Accessibility improvement; we shouldn't skip levels. I called this out here: https://github.com/PrairieLearn/PrairieLearn/pull/12496#discussion_r2261385803